oauth/connexion.php

```diff
@@ -33,7 +33,7 @@
    		// Remove every html tag and useless space on username (to prevent XSS)
    	   	$user=strip_tags(trim($_POST['user']));
 
-    	$user=$_POST['user'];
+    	$user=strtolower($_POST['user']);
```

Comment:
Note: I'm unclear why $user is being set to the result of `strip_tags(trim())` and immediately re-set to the POST value, but I didn't want to try and make both changes at once since that's not quite relevant to this particular change.